web/src/views/ProfileCreate.tsx

```diff
@@ -113,4 +161,19 @@ const ProfileCreate: React.FC = () => {
   );
 };
 
-export default ProfileCreate;
+const mapStateToProps = (state: any, githubID: any) => ({
+  technicalLead1: selectTechnicalLead(0)(state),
+  technicalLead2: selectTechnicalLead(1)(state),
+  stateProductOwner: selectProductOwner()(state),
+});
+// const mapDispatchToProps = (dispatch: any) => ({
```

Comment:
I'm assuming you won't be using this commented code for anything, right? It should be left out unless it's necessary.